src/components/Cell/Cell.test.tsx

```diff
@@ -20,16 +20,19 @@ describe('Cell', () => {
     [undefined, 'undefined'],
     [[1, 2, 3], '[1,2,3]'],
   ])('renders the value (%s) as string by default: %s', (value, text) => {
-    const { getByText } = render(<table>
-      <tbody>
-        <tr>
-          <Cell
-            cell={{ value }}
-            {...rest}
-          ></Cell>
-        </tr>
-      </tbody>
-    </table>)
+    const { getByText } = render(
+      <table>
+        <tbody>
+          <tr>
+            <Cell
+              cell={{ value }}
+              {...rest}
+            >
+            </Cell>
```

Comment:
```
/>
```

is there a lint rule for "use self-closing tag for empty cells"?

Reply:
yes: https://eslint.style/rules/jsx-self-closing-comp